src/org/usfirst/frc/team1076/robot/subsystems/Claw.java

```diff
@@ -0,0 +1,38 @@
+package org.usfirst.frc.team1076.robot.subsystems;
+
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class Claw extends Subsystem {
+	
+	SpeedController clawMotor;
+	
+	public Claw(SpeedController clawMotor) {
+		this.clawMotor = clawMotor;
+	}
+	
+	public void setMotorSpeed(double speed) {
+		this.clawMotor.set(speed);
+	}
+    
+	public void intakeBall() {
+		setMotorSpeed(1);
+	}
+	
+	public void expellBall() {
+		setMotorSpeed(-1);
+	}
+	
+	public void stop() {
+		setMotorSpeed(0);
+	}
+
+    public void initDefaultCommand() {
```

Comment:
Since there only needs to be one `IO` instance, you could just create it within the command itself (make some kind of factory method). The commands can just poll directly from that object instead. That would let you do`setDefaultCommand(new ControlClawWithTeleop())` and not have to worry about how the command would work.

You could also pass explicit IO and Subsystem instances, but that seems kind of messy and requires too many references to just a single instance.